### ISSUE TITLE
Unable to connect v1.3.0-beta.5

### ISSUE THREAD

**Issue by dsiminiuk** [marcelklehr/floccus#35]
Great idea!

However, my nextcloud server is local on my LAN and I cannot connect on port 8443.

cloud.localdomain.com:8443
cloud.localdomain.com:8443/login
https://cloud.localdomain.com:8443
https://cloud.localdomain.com:8443/login

no joy.

**Comment by marcelklehr** [comment]
Hey there,
Let's do some troubleshooting (gotta love that word).
First up: nextcloud server version and bookmarks app version please :)

**Comment by marcelklehr** [comment]
ah, but on the server you have also installed the "bookmarks" nextcloud app, right? I also need that version. If you have not installed that app, install it; problem solved ;)

**Comment by dsiminiuk** [comment]
Yes that would be the initial problem. Not being familiar with the Nextcloud bookmarks app, it isn't really clear here that it is a pre-requisite. Ignorance is not bliss. :)

After installing the Nextcloud bookmarks app I was able to connect, and I forced a sync which appears to have completed.

I now have an "Other bookmarks" on my Chrome bookmarks bar and when I click on that I see my account listed and when I click on that it says "(empty)" and I see no bookmarks in the Nextcloud boookmarks area (yet). Is this the 25 minute delay?
